src/main/java/com/teamrocket/tms/models/dtos/UserDTO.java

```diff
@@ -23,9 +24,7 @@ public class UserDTO {
     @Size(min = 3, max = 30, message = "must be between 3 and 30 characters")
     private String lastName;
 
-    @NotBlank
-    @Size(min = 3, max = 30, message = "must be between 3 and 30 characters")
-    private String role;
+    private RoleEnum roleEnum;
```

Comment:
Usually the naming convention for enum properties doesn't contain the "Enum" ending